cmake/linux/build-installer

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -e
+BINDIR="$1"
+RUNFILE="$2"
+[ "$BINDIR" -a "$RUNFILE" ]
+
+cd "$BINDIR"
+[ -e lmmsversion.h ]
+VERSION=$(grep "LMMS_VERSION " lmmsversion.h | cut -d \" -f 2)
```

Comment:
Albeit more complicated to edit, I'd recommend we [configure this file like we do with our other scripts](https://github.com/LMMS/lmms/blob/master/cmake/apple/install_apple.sh.in), rather than parsing the header file.